### PR TITLE
Header component

### LINKED ISSUE #5: As developer i want create a header component, so that my app have a header
header

### PR DESCRIPTION
**Pull request by dimas1976**
https://mentalika-headercompone-pqr3b6.herokuapp.com/storybook/?path=/story/component-header--app-header

fix #5

### CODE REVIEW

File: src/app/components/assets/BrainLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function BrainLogo(): JSX.Element {
+  return (
+    <svg
+      width="30"
+      height="30"
+      viewBox="0 0 30 30"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (marwinburesch) [inline src/app/components/assets/BrainLogo.tsx:12]: Why not pass props, you might want to use the logo at a different size later :)

File: src/app/globals.css [hunk 2]
```diff
@@ -44,6 +44,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100vh;
```
Comment (marwinburesch) [inline src/app/globals.css]: I'd set this somewhere in `App.tsx`

**Review by marwinburesch — COMMENTED** [review]
Nice🍨

File: src/app/components/assets/BrainLogo.tsx (hunk 1 above)
Comment (dimas1976) [reply src/app/components/assets/BrainLogo.tsx:10]: can i adjust svg size with css?

File: src/app/globals.css (hunk 2 above)
Comment (dimas1976) [reply src/app/globals.css]: done

File: src/app/components/assets/BrainLogo.tsx (hunk 1 above)
Comment (dimas1976) [reply src/app/components/assets/BrainLogo.tsx:12]: done

**Review by marwinburesch — COMMENTED** [review]
I like it, straight to the point 🦾. Although the logo has very little space above it. Might get fixed once the component is implemented though. :)
I am sure you meant "fixes #5 "

**Pull request merged**